### PR TITLE
[INLONG-2588][Manager] Support cluster management

### LINKED ISSUE #2588: [Feature][Manager] support cluster management
### Description

support `manager` module cluster management

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by herywang**
Fixes #2588

### Motivation

*Adding some method to support cluster management.*

### Modifications
None.

### CODE REVIEW

**Review by healchow — APPROVED** [review]

**Comment by dockerzhang** [comment]
@herywang please do not remove the `.idea/vcs.xml` file, it's useful to link the valid issues or PRs.

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java [hunk 1]
```diff
@@ -58,6 +59,37 @@
         return Response.success(clusterInfoService.list(request));
     }
 
+    @RequestMapping(value = "/common/save", method = RequestMethod.GET)
```
Comment (dockerzhang) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java:62]: what does the `common` stand for? it's better to use an explicit name.

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java [hunk 2]
```diff
@@ -58,6 +59,37 @@
...
+    }
+
+    @RequestMapping(value = "/common/get/{id}")
+    @ApiOperation(value = "Query cluster information of the common")
+    @ApiImplicitParam(name = "id", value = "common cluster ID", dataTypeClass = Integer.class, required = true)
+    public Response<ClusterInfo> getCluster(@PathVariable Integer id) {
+        return Response.success(clusterInfoService.get(id));
+    }
+
+    @RequestMapping(value = "/common/update", method = RequestMethod.POST)
```
Comment (dockerzhang) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java:77]: ditto

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java (hunk 1 above)
Comment (herywang) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java:62]: **I will modify this problem in the next issue and pull request.**

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java [hunk 3]
```diff
@@ -58,6 +59,37 @@
...
 
+    @RequestMapping(value = "/common/save", method = RequestMethod.GET)
+    @ApiOperation(value = "Add a cluster info")
+    @OperationLog(operation = OperationType.CREATE)
+    public Response<Integer> saveCluster(@RequestBody ClusterInfo clusterInfo) {
+        String currentUser = LoginUserUtil.getLoginUserDetail().getUserName();
+        return Response.success(clusterInfoService.save(clusterInfo, currentUser));
+    }
+
+    @RequestMapping(value = "/common/get/{id}")
```
[the remark this answers is not in the document]
Comment (herywang) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java:70]: I will modify this problem in the next issue and pull request.

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java (hunk 2 above)
Comment (herywang) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java:77]: I will modify this problem in the next issue and pull request.

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**